app/config/config.yml

```diff
@@ -20,6 +20,7 @@ parameters:
 framework:
     #esi:             ~
     #translator:      { fallbacks: ["%locale%"] }
+    #csrf_protection: ~
```

Comment:
This late in the game, we can leave it for later, if y'all want. Since it is in a .yml file, it gets ignored by GCS anyway. I'm fine either way actually.